printer_mods/README.md

```diff
@@ -174,6 +174,7 @@ like so:
 |Majarspeed| [Hartk board mount galileo](./Majarspeed/Hartk_Mount_Galileo) | It hangs stuff on your tool | :x: | :x: | :heavy_check_mark: | :x: |
 |madfabrications |[Vertical PSU Mount](./madfabrications/Vertical_PSU_Mount) |Mounts the PSU vertically on the DIN rails|:x: |:x: |:heavy_check_mark:|:x:|
 |madros |[V0 OLED-skirt](./madros/V0_skirt_OLED) |V0 skirt with 0.96" OLED|:heavy_check_mark: |:x: |:x: |:x:|
+|Maramao| [BTT piTFT43 Mount](.\maramao\STL\BTT_piTFT43_mount) | Mount for the BTT piTFT43 screen | :x: | :x: | :heavy_check_mark: | :grey_question: |
```

Comment:
This link is broken - the slashes are backwards and the directory is incorrect.  If you move your mod to `printer_mods/maramao/BTT_piTFT43_mount/` as suggested in my other comment, then your link here would be

    [BTT piTFT43 Mount](./maramao/BTT_piTFT43_mount)

Also please add a question mark or an X in the last column (Trident) of the compatibility matrix.